### ISSUE TITLE
[Question] Now Playing API endpoint

### ISSUE THREAD

**Issue by SamJongenelen** [jellyfin/jellyfin#9665]
Hey guys, I've been using Jellyfin with a lot of pleasure. I've connected my HomeAssistant to the REST API of JF, and was surprised to find only the `sessions` endpoint exists to gather info on who is now playing.  Of course, I was able to filter the Session response, but still I found it hard to combine different states (e.g. nothing playing but active session, playback started') and also, (the response is enormous :) ) into a nice sensor in HomeAssistant.

As I am a dotnet developer myself, I was wondering if a endpoint like `sessions/nowplaying`, that would return a simple object with something like Username, NowPlaying item name, would be accepted the code base. There might be a reason it isn't already. If it is oké - I can probably make it.

Basically the implementation could filter and project output of the `SessionManager`

**Comment by crobibero** [comment]
Personally I don’t see the use in creating a simpler endpoint if all of the information is already returned by an existing (but complex) endpoint.

**Comment by nielsvanvelzen** [comment]
Adding a new parameter to the existing endpoint to filter only sessions currently playing media might be a nice addition.

**Comment by SamJongenelen** [comment]
Yeah i agree it maybe shouldn't be added, hence i ask here. 

Regarding the extra param: my 'problem' lies with the complex return type. It's hard to create a single HomeAssistant rest sensor that answers my question 'who is playing, what are they playing, and what is the bandwith consumption (based on content info)'. This is because the return object is complex and pretty big (it is correctly filled of course)

**Comment by nielsvanvelzen** [comment]
Doesn't the Jellyfin integration in Home Assistant provide this information already?
https://www.home-assistant.io/integrations/jellyfin/

**Comment by SamJongenelen** [comment]
Unfortunately not, it provides a media_player for all sessions as additional functionality indeed. I think i could use this, but I dont want the primary function of the integration - to control media - in HomeAssistant, I use it for network

**Comment by nielsvanvelzen** [comment]
Our API is already quite bloated. Personally I think we shouldn't add another endpoint if an existing one already provides the necessary information (although with some extra bytes).

Extending the endpoint to allow additional filtering should be fine, but changing the response to return less data purely because that makes it easier to work with in Home Assistant feels like the wrong approach to me.

**Comment by SamJongenelen** [comment]
Thanks for the feedback :) I personally think the SessionController isn't that big - but yes it is another addition.

Regarding the ' for home assistant '  - in Jellyfin Admin page i also wouldn't mind seeing only 'now playing' sessions.
